calico-enterprise/getting-started/bare-metal/troubleshoot.mdx

```diff
@@ -0,0 +1,88 @@
+---
+description: Troubleshoot non-cluster hosts and VMs setup
+---
+
+# Troubleshoot non-cluster hosts and VMs setup
+
+This document provides guidance for troubleshooting non-cluster host setup for hosts and VMs.
+
+## Useful commands
+
+These commands can help you collect logs and monitor system activities during troubleshooting.
+
+### On non-cluster hosts or VMs
+
+```bash
+journalctl -xue calico-node.service -f
+journalctl -xue calico-fluent-bit.service -f
+```
+
+### On the cluster side
+
+```bash
+kubectl logs -n calico-system -l k8s-app=calico-typha-noncluster-host
+kubectl logs -n tigera-manager -l k8s-app=tigera-manager -c tigera-voltron
+```
+
+Monitor CertificateSigningRequests (CSR):
```

Comment:
You could briefly elaborate why this can be helpful.

Reply:
I've added some details about why monitoring CSR is useful for debugging. Please take another look.